core/loadBalancer/src/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -50,6 +41,7 @@ class LoadBalancerService(config: WhiskConfig, verbosity: Verbosity.Level)
 
     /** The execution context for futures */
     implicit val executionContext = ExecutionContextFactory.makeExecutionContext()
```

Comment:
Why make the execution context then the actor system? Either pass the context to the system, or create the system first and get the context out of it. Both options will ensure the context is shared, currently it's not guaranteed.